app/routes/base.py

```diff
@@ -14,9 +14,9 @@ class UserResources:
 
 def get_user_resources():
     user_resources = {
-        "Coins": UserResources(current_user.army.coin, 'coin'),
-        "Metal": UserResources(current_user.army.metal, 'metal'),
-        "Wood": UserResources(current_user.army.wood, 'wood'),
+        "coin": UserResources(current_user.army.coin, 'coin'),
```

Comment:
I would do it differently.
I would have a simple list of resources, and then have a function in the user model to do a for loop on that list and return the resources like that

Reply:
this is a old list that we had, all i did is to change the key names so i will be able to use it in the html page